NEWS.md

```diff
@@ -2,8 +2,43 @@
 
 ## v18
 
-- `$SCRIPT` was renamed in favor of `$CHROOT_SCRIPT`. `$SCRIPT` can
-  still be used but is considered deprecated.
+- `$SCRIPT` was renamed to `$CHROOT_SCRIPT`. `$SCRIPT` can still be used
+  but is considered deprecated.
+- Added `RuntimeTrees=` setting to mount directories when booting images
+  via `mkosi boot`, `mkosi shell` or `mkosi qemu`. The directories are
+  mounted with a uid map that maps the user invoking mkosi to the root
+  user so that all files in the directory appear as if owned by the root
+  user in the container or virtual machine and any new files created in
+  the directories are owned by the user invoking mkosi. To make this
+  work in VMs, we use `VirtioFS` via `virtiofsd`.
+- Added support for booting directory images with `mkosi qemu` via
+  `VirtioFS`. When `CONFIG_VIRTIOFS` and `CONFIG_VIRTIO_PCI` are builtin
+  modules, no initramfs is required to make this work.
+- Added `Include=` or `--include` to include extra configuration files
+  or directories.
+- Added support for specifiers to access the current value of certain
+  settings during configuration file parsing.
+- `mkosi` will now exit with an error when no configuration was
+  provided.
+- Multiple scripts of the same type are now supported.
+- Custom distributions are now supported via the new `custom`
+  distribution. When using `custom` as the distribution, the rootfs must
+  be provided via base trees, skeleton trees or prepare scripts.
+- We now use local GPG keys for rpm based distributions if the
+  `distribution-gpg-keys` package is installed on the host.
+- Added `RuntimeSize=` to grow the image to a specific size before
+  booting it when using `mkosi boot` or `mkosi qemu`.
+- We now set `MKOSI_UID` and `MKOSI_GID` when running scripts which are
+  set to the uid and gid of the user invoking mkosi respectively. These
+  can be used to run commands as the user that invoked mkosi.
+- Added an `Architecture=` match
+- Initrds specified with `Initrds=` are now used for grub menuentries as
+  well.
+- `IMAGE_ID` and `IMAGE_VERSION` are now written to os-release if
```

Comment:
```suggestion
- `ImageId=` and `ImageVersion=` are now written to os-release if
```
to make it clear that that's enough and people don't need to set the environment variables (which we already set).